torch/onnx/symbolic_opset8.py

```diff
@@ -74,6 +74,22 @@ def symbolic_fn(g, input, output_size, align_corners=None):
 upsample_trilinear3d = _interpolate('upsample_trilinear3d', 5, "linear")
 
 
+def __interpolate(g, input, size, scale_factor, mode , align_corners):
+    align_corners = sym_help._maybe_get_const(align_corners, 'b')
+    if not align_corners.node().mustBeNone() and align_corners:
+        return _unimplemented("interpolate", "align_corners == True")
+
+    if not scale_factor.node().mustBeNone() and sym_help._is_value(scale_factor):
```

Comment:
Can we extract this to some helper function? So we don't need to repeat this pattern manually.

Reply:
Feel free to do it in a dedicated refactor PR.